.changeset/silver-buckets-hope.md

```diff
@@ -0,0 +1,6 @@
+---
+"@atproto/lex-cli": minor
+"@atproto/xrpc-server": patch
+---
+
+Updated to use refactored version of `@atproto/xrpc`.
```

Comment:
If this is truly just a dependency update then we don't need a changeset for it, as changesets will detect this and add a line to the changelog.  E.g. like this: https://github.com/bluesky-social/atproto/blob/main/packages/dev-env/CHANGELOG.md#0338

Reply:
Ah that's right, the `lex-cli` changes are more in-depth and are an actual version bump.